### PR TITLE
Extend Audio Settings to use OS Audio Management Systems

### LINKED ISSUE #2069: [Enhancement]: Extend Audio Settings to use OS Audio Management Systems
### Describe the enhancement

Extend the audio control in the Stereum settings to use the OS Audio Management System, making them controllable via f.e. Audio Mixer in Windows.

image

### Dependencies

--

### Additional Context

--

### Acknowledgment

- [X] I have reviewed the existing enhancements and confirm that this suggestion is unique.

### PR DESCRIPTION
**Pull request by mabasian**
FIX #2069

### CODE REVIEW

**Review by MaxTheGeeek — APPROVED** [review]
lg

File: launcher/src/components/UI/the-control/ControlAlert.vue [hunk 1]
```diff
@@ -545,10 +550,26 @@ export default {
...
+    },
+
+    async syncSystemVolume() {
+      try {
+        const systemVolume = await window.promiseIpc.getVolume();
+
+        this.currentVolume = systemVolume;
+        this.volState = systemVolume > 0;
+      } catch (error) {
+        console.error("Failed to sync system volume:", error);
```
Comment (NeoPlays) [inline launcher/src/components/UI/the-control/ControlAlert.vue:571]: duplicated code,
move this into a composable, or do it globally

File: launcher/src/preload.js [hunk 2]
```diff
@@ -11,4 +11,8 @@ contextBridge.exposeInMainWorld("promiseIpc", {
     ipcRenderer.on("terminal-output", (_, arg) => callback(arg));
     return () => ipcRenderer.removeListener("terminal-output", callback);
   },
+
+  // Audio control functions
+  setVolume: (volume) => ipcRenderer.invoke("set-system-volume", volume),
+  getVolume: () => ipcRenderer.invoke("get-system-volume"),
```
Comment (NeoPlays) [inline launcher/src/preload.js:17]: this is not right handle it like we always did in `ControlService.js` and don't access 'window.promiseIpc` from frontend.

**Review by NeoPlays — COMMENTED** [review]
i don't think this should have been merged like this

**Pull request merged**